Common/src/main/java/com/tabnineCommon/chat/TabnineChatProjectManagerListener.kt

```diff
@@ -36,6 +38,15 @@ class TabnineChatProjectManagerListener private constructor() : ProjectManagerLi
     fun start() {
         if (initialized.getAndSet(true)) return
 
+        if (StaticConfig.getTabnineEnterpriseHost().isPresent) {
```

Comment:
also, not critical, but i'd consider creating a function that accepts a function: `invokeWhenChatEnabled`, that'll check if isOnprem or capability enabled. Then you use it like so:
```kotlin
invokeWhenChatEnabled {
  ProjectManager.getInstance().openProjects.forEach { projectOpened(it) }
  ApplicationManager.getApplication().messageBus
      .connect(this).subscribe(ProjectManager.TOPIC, this)
}
```